guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -11,13 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>replace with Table of <strong>C</strong>ontents</p><h2><br></h2><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>